### DISCUSSION TITLE
Take screenshot in navigation / follow

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#95] [Q&A] [answered]
Today I had a problem when resuming a tour. But first I want to post this as a handling issue

To take a screenshot during navigation is too complicated, too much fingertips and then it asks to confirm the saving of the png file, nothing that you do while driving. 

I would propose this to make also as a 1 fingertip feature via an action menu and then automatically save as png file in a subfolder of the cruiser path, then to report a problem while navigation it is much easier to document this.

**Comment by devemux86** [comment]
Thanks for the report, I agree that the quick screenshot function can be improved!

The issue is that Android does not longer allow to write / read external app locations.
Must open Android browser so users select the location > giving storage permission.

Automatic write, like save a screenshot, means we have to use an internal app folder.
Could be the internal "media" folder that is accessible by file managers / other apps.

However then users would need to find their screenshots later in that internal folder
or the app needs a manager for the stored screenshots?

**Reply by 0709wiwiwi** [reply]
A13.
Preferably only use Android Android >media > gr.talent.cruiser > Screenshots.
Through the Cx File Explorer create a shortcut "Add to Home screen". 
Notice how created shortcut works.
![Screenshots in media](https://user-images.githubusercontent.com/54167201/227326303-f6b87080-00f0-4c14-9e84-1bab1147e56f.gif)

**Reply by devemux86** [reply]
> Using the default file explorer (as invoked by Cruiser), I cannot access Android data.

Android/data can only be accessed by some file managers. Not applications (via Android browser).
Android/media is currently more accessible.

**Reply by WolfgangZ42** [reply]
see https://github.com/devemux86/cruiser/discussions/57#discussioncomment-5101711 - I'd spend the short tap on compass for direct screenshots ...

**Comment by devemux86** [comment, marked as answer]
Let's discuss all quick actions during navigation / follow in:
- https://github.com/devemux86/cruiser/discussions/405
